e2e_samples/fabric_dataops_sample/README.md

```diff
@@ -106,6 +106,10 @@ Here is a list of resources that are deployed:
   - Contributor permissions to an Azure Repo in such Azure DevOps environment.
   - A branch and a folder in the repository where the Fabric items will be committed. The folder must already exist.
 
```

Comment:
Please check why this hasn't updated the TOC.

Reply:
Good catch! Ah, i had disabled my markdown all in one extension because of some issues i was having with it. Just updated the toc